yb-voyager/cmd/root.go

```diff
@@ -96,18 +97,19 @@ func startPprofServer() {
 	*/
 }
 
+var noLockNeededList = []string{
+	"yb-voyager version",
+	"yb-voyager import data status",
+	"yb-voyager export data status",
+	"yb-voyager cutover status",
+	"yb-voyager fall-forward status",
+	"yb-voyager cutover initiate",
+	"yb-voyager fall-forward switchover",
+	"yb-voyager end",
+	"yb-voyager end migration",
+}
+
```

Comment:
this list needs to be modified with this #1187. cc: @makalaaneesh

Reply:
Correct. 
Get rid of 
- yb-voyager fall-forward status
- yb-voyager cutover initiate
- yb-voyager fall-forward switchover

Add:
- yb-voyager initiate cutover to target/source/source-replica

Reply:
cool.
Also need to remove `"yb-voyager cutover status"` then.

Reply:
so this is the new list: 
```

var noLockNeededList = []string{
	"yb-voyager version",
	"yb-voyager import data status",
	"yb-voyager export data status",
	"yb-voyager initiate cutover to source",
	"yb-voyager initiate cutover to target",
	"yb-voyager initiate cutover to source-replica",
	"yb-voyager end",
	"yb-voyager end migration",
}
```

Reply:
cc @priyanshi-yb @makalaaneesh ^^